apps/emqx_bridge/src/emqx_bridge_api.erl

```diff
@@ -913,7 +913,7 @@ format_resource(
     redact(
         maps:merge(
             RawConfFull#{
-                type => downgrade_type(Type, RawConf),
+                type => downgrade_type(Type, emqx_bridge_lib:get_conf(Type, BridgeName)),
```

Comment:
Small suggestion:

```suggestion
                type => downgrade_type(ActionType, emqx_bridge_lib:get_conf(ActionType, BridgeName)),
```

Reply:
Well ... it can be both. For non v2 types it still is a bridge type. But I guess eventually this will go away. But then we should also remove all those `is_bridge_v2_type` checks.